changelog/seqera-enterprise/v24.2_major.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Seqera Enterprise v24.2_major
+date: 2024-12-10
+tags: [seqera enterprise]
+---
+
+Seqera Enterprise version 24.2 introduces a number of new features to Data Studios, the addition of the global Nextflow config, as well as a number of bug fixes and performance enhancements.
+
+## New features and improvements
+
+### Nextflow
+
+  - Added global Nextflow config: This provides the ability to define the Nextflow config in the CE as a "global" configuration. The configuration is defined in the pipeline config and at launch level and can override the one defined in the CE config.
+
+### Launchpad
+
+- The new [Launch Form](https://docs.seqera.io/platform/24.2/launch/launchpad#launch-form) is enabled by default. If you've activated the new launch form already using the environment variable `TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES`, nothing will change. If the launch form has not been activated, it will now show by default. You can disable this per workspace, or across your entire deployment using environment variables. When the environment variable is `null` (undefined), no workspaces are enabled. When it's left empty, all are enabled.
+- Implemented custom launch container logic to specify a custom container to run Nextflow when submitting a workflow execution.
```

Comment:
@llewellyn-sl, I'm not sure this belongs here but also not sure where else it would go. Also want to link to https://docs.seqera.io/platform/24.2/enterprise/advanced-topics/custom-launch-container if that's the correct link.